tempesta_fw/connection.h

```diff
@@ -177,7 +177,9 @@ tfw_connection_put(TfwConnection *conn)
 static inline void
 tfw_connection_put_to_death(TfwConnection *conn)
 {
-	atomic_add(TFW_CONN_DEATHCNT, &conn->refcnt);
+	/* Do not put dead connections to death state */
+	if (tfw_connection_nfo(conn))
```

Comment:
As far as I understand, initially `refcnt` is zero. Currently it's not assigned a value until a connection is established at least once.